guides/V1_Meeting.md

```diff
@@ -0,0 +1,79 @@
+---
+copyright: 'Copyright IBM Corp. 2018'
+link: 'meeting'
+is: 'experimental'
+---
+
+# Get meeting details
+
+## Concepts
+
+The _meeting_ query allows the API caller to get details about the meeting.  The query accepts a Space ID (spaceId) and will return meeting details if the calling user has access to the meeting.  The calling user will have access to the meeting if the user has access to the space which contains the meeting, or the meeting has been marked to allow public participants.
+
+## Schema
+
+### Meeting Query
+
+
+```graphql
+type QueryRoot {
+  ...
+  meeting(spaceId: ID!): Meeting
+}
+
+type Meeting {
+  active: Boolean!
+  meetingNumber: String
+  password: String
+  allowPublic: Boolean!
+  startTime: Date
+  joinInfo: JoinInfo
+}
+
+type JoinInfo {
+  meetingProviderJoinUrl: String
+  pstnPasscode: String
+  dialInNumbers : [DialInNumber]
+}
+
+type DialInNumber {
+  countryCode: String!
+  number: String!
+}
+
+```
+
+## Example Request
+
+~~~~
+Method: POST
+URL: https://api.watsonwork.ibm.com/graphql
+Headers: 'Content-Type: application/graphql' , 'x-graphql-view: PUBLIC, EXPERIMENTAL'
+Body:
+{
+  query {
+    meeting(spaceId: "5a9ee86ee4b0cb1ca3dfef33") {
```

Comment:
There is a convention used in the docs for using "space-id" as a placeholder ID I think. I think the use of the variables section might be better if we explained it up front and did it consistently, but even then I think we'd need a placeholder space ID to serve as an example. I think it's better to use an obvious placeholder like space-id rather than a real ID.